Rename wxListCtrl::HasCheckboxes() and EnableCheckboxes()

discnl opened the pull request:
Rename them to HasCheckBoxes and EnableCheckBoxes for consistency with
wxCheckBox naming.

See also PR https://github.com/wxWidgets/wxWidgets/pull/153

I made this a PR because I wasn't certain about how to mention the changes in docs/changes.txt.

MaartenBent commented:
What about the private functions (```IsClickInsideCheckbox()```, and some in `listtest.cpp`)?

discnl commented:
Thanks! I didn't notice the commit for the generic implementation of listctrl checkboxes before.

I had left the listctrl sample alone because I thought there were also earlier references to Checkbox there, which I wanted to change as part of a rename for all private (sample) methods. However all Checkbox references were new in listtest.cpp so I added those as well now.

discnl committed:
Rename wxListCtrl::HasCheckboxes() and EnableCheckboxes()

Rename them to HasCheckBoxes and EnableCheckBoxes for consistency with
wxCheckBox naming.

See also PR https://github.com/wxWidgets/wxWidgets/pull/153

discnl committed:
Rename wxListMainWindow::IsInsideCheckBox to IsInsideCheckbox

discnl committed:
Rename methods in listctrl sample to use CheckBox instead of Checkbox
